packages/discord.js/typings/index.d.ts

```diff
@@ -955,6 +955,7 @@ export class Client<Ready extends boolean = boolean> extends BaseClient {
   private _eval(script: string): unknown;
   private _validateOptions(options: ClientOptions): void;
   private get _censoredToken(): string | null;
+  private readonly _ready: Ready;
```

Comment:
A comment would be ideal as to why this is needed and not present in JavaScript like so:

https://github.com/discordjs/discord.js/blob/c50809e20648cacea99f5450e8073d960ff8aa39/packages/discord.js/typings/index.d.ts#L1799-L1800